### PR TITLE
fix(FileNotFoundError): Display proper error message for fileNotFoundError

### LINKED ISSUE #79: Invalid File Path in Atarashi
**Whenever a invaid file path is provided to atrashi it generate the following error:**
env) akshay@akshay-VirtualBox:~/atarashi/atarashi/evaluator$ atarashi -a tfidf Testfiles/APSL-style.html
Traceback (most recent call last):
File "/home/akshay/atarashi/env/bin/atarashi", line 8, in
### sys.exit(main())
File "/home/akshay/atarashi/env/lib/python3.8/site-packages/atarashi/atarashii.py", line 123, in main
result = atarashii_runner(inputFile, processedLicense, agent_name, similarity, ngram_json, verbose)
File "/home/akshay/atarashi/env/lib/python3.8/site-packages/atarashi/atarashii.py", line 83, in atarashii_runner
result = scanner.scan(inputFile)
File "/home/akshay/atarashi/env/lib/python3.8/site-packages/atarashi/agents/tfidf.py", line 140, in scan
return self.__tfidfcosinesim(filePath)
File "/home/akshay/atarashi/env/lib/python3.8/site-packages/atarashi/agents/tfidf.py", line 112, in __tfidfcosinesim
processedData1 = super().loadFile(inputFile)
File "/home/akshay/atarashi/env/lib/python3.8/site-packages/atarashi/agents/atarashiAgent.py", line 44, in loadFile
self.commentFile = CommentPreprocessor.extract(filePath)
File "/home/akshay/atarashi/env/lib/python3.8/site-packages/atarashi/libs/commentPreprocessor.py", line 129, in extract
data1 = licenseComment(data)
File "/home/akshay/atarashi/env/lib/python3.8/site-packages/atarashi/libs/commentPreprocessor.py", line 42, in licenseComment
for id, item in enumerate(data[0]["multi_line_comment"]):
IndexError: list index out of range
(env) akshay@akshay-VirtualBoxatarashi/atarashi/evaluator$

**### Instead It should Generate a simple error msg that the file path that was provided was wrong**

### PR DESCRIPTION
**Pull request by Aman-Codes**
Closes #79

While testing I also found some extra error logs which occurs when we specify incorrect similarity parameter to tfidf agent. 

# Changes Made
- Added proper error message for fileNotFoundError
- Removed extra error logs on specifying incorrect similarity parameter to agents

# Screenshots
## Before

fileNotFoundErrorBefore

similarityErrorBefore

## After

fileNotFoundErrorAfter

similarityErrorAfter

### CODE REVIEW

File: atarashi/atarashii.py [hunk 1]
```diff
@@ -120,25 +125,29 @@ def main():
...
+                "sim_score": 1,
+                "sim_type": "dld",
+                "description": ""
+            }]
+      result = list(result)
+      result = {"file": os.path.abspath(inputFile), "results": result}
+      result = json.dumps(result, sort_keys=True, ensure_ascii=False, indent=4)
+      print(result + "\n")
+  except FileNotFoundError:
+    print("FileNotFoundError: File does not exist. Enter a valid file path")
```
Comment (GMishx) [inline atarashi/atarashii.py:150]: What if we print the file name from exception like `print("Error: " + e.strerror+ ": '" + e.filename + "'")` ?

Comment (Aman-Codes) [reply atarashi/atarashii.py]: I tried changing line 149 and 150 to
```
except FileNotFoundError as e:
print("Error: " + e.strerror+ ": '" + e.filename + "'")
```
It is running fine locally but the automated tests are failing. Can you guide me why ?

Comment (GMishx) [reply atarashi/atarashii.py]: The changes with Nirjas were merged and a new release was tagged. They are breaking atarashi.
I have created PR #81 to fix it. /cc @ag4ums

**Review by GMishx — APPROVED** [review]
Changes looks good. Working as expected.

**Pull request merged**